examples/python-async/receive_data.py

```diff
@@ -0,0 +1,17 @@
+import asyncio
+
+from dora import Node
+
+
+async def main():
+    node = Node()
+    for _ in range(100):
```

Comment:
I am wondering if we can get to something like this:

```py 
async for event in node:
  # ...
```

Reply:
Sure thing!

Reply:
Ok, so it seems that `__anext__` the async iterator is kind of hard to implement so i'm going to merge this like this for now. And maybe come back on this once the documentation on pyo3 is better.

See: https://github.com/PyO3/pyo3/issues/1632#issuecomment-2747696822

Reply:
Of course. This is awesome!